### ISSUE TITLE
DropdownItem is not reachable with keyboard tab -Firefox

### ISSUE THREAD

**Issue by anirudhaBapat** [carbon-design-system/carbon#351]
## DropdownItem is not reachable with keyboard tab: When I tried putting in keyboard nav for the Dropdown, DropdownItems are not reachable with tab

> Dropdown 

> What did you expect to happen?
It should focus-able/reachable using tab *in all browsers* 

>What happened instead?
It only works with Chrome not with Firefox/Safari (IE have not tested)

> What browser are you working in? 
Firefox, Safari

> What version of the Carbon Design System are you using? 
7.23.1

**Comment by asudoh** [comment]
@anirudhaBapat Quickly tried http://carbondesignsystem.com/components/dropdown/code site with Firefox 52.3.0 ESR, like:

* Going to Dropdown with Tab
* Hit Enter (To show dropdown items)
* Hit Tab several times - I can navigate through dropdown items
* Hit Shift+Tab several times - I can navigate back

So I don’t see a problem there?

**Comment by marijohannessen** [comment]
Closing this issue due to lack of response from issue creator. Please feel free to re-open if needed.
